kyo-tag/shared/src/main/scala/kyo/tags.scala

```diff
@@ -307,8 +294,8 @@ object Tag:
                     val sym = tpe.typeSymbol
                     val path = tpe.dealias.baseClasses.map { sym =>
                         val name = toCompact.getOrElse(sym.fullName, sym.fullName)
-                        val size = encodeInt(name.length())
-                        val hash = encodeInt(Math.abs(name.hashCode()) % charsPrintable.length)
+                        val size = IntEncoder.encode(name.length())
+                        val hash = IntEncoder.encodeHash(name.hashCode())
```

Comment:
Will this be a problem for custom hashcodes?

Reply:
Not sure I understood. Do you mean using the encoder in other places besides tags?